data/json/martialarts.json

```diff
@@ -936,6 +936,77 @@
             "tec_venom_snake_bite",
             "tec_venom_snake_strike"
         ]
+    },{
+        "type" : "martial_art",
+        "id" : "style_hema",
+        "name" : "HEMA",
+        "description": "The modern recreation of European fencing, based off the manuals of Medieval and Renaissance fencing masters. It emphasises quick footwork and aggressive striking. Skilled fencers can apply their techniques to make lightning fast attacks and devastating combos, while maintaining their defense.", 
+        "arm_block" : 99,
+        "leg_block" : 99,
+        "onmove_buffs" : [
+            {
+                "id" : "hema_move_buff",
+                "name" : "Quick Footwork",
+                "description" : "You keep your distance from the enemy, giving you +1 bonus dodge and +2 to your dodge skill, as long as you keep moving",
```

Comment:
Hmm, that's a fairly nice passive bonus.  I'd sought to have Niten Ichi-Ryu make the user *more* vulnerable to groups, not less.  (MA styles have been suffering from power creep.)